zerver/webhooks/buildbot/doc.md

```diff
@@ -0,0 +1,24 @@
+Get Zulip notifications for your Buildbot builds!
+
+1. {!create-stream.md!}
+
+1. {!create-a-bot-indented.md!}
+
+1. Edit the Buildbot configuration file to add a new Zulip reporter
+ ([or follow the steps listed here][1]):
+
+        from buildbot.plugins import reporters
+
+        zs = reporters.ZulipStatusPush(endpoint='{{ zulip_url }}',
```

Comment:
Hmm, why is this not `api_url`?  I think either we want to rename `endpoint` to `server_url` or we want this to have the value of `api_url`, since "endpoint" generally means a full path, not just the host part.

I probably prefer changing the variable name.